Lib/argparse.py

```diff
@@ -2098,6 +2098,11 @@ def consume_positionals(start_index):
             # and add the Positional and its args to the list
             for action, arg_count in zip(positionals, arg_counts):
                 args = arg_strings[start_index: start_index + arg_count]
+                # Strip out the first '--' if it is not in PARSER or REMAINDER arg.
```

Comment:
Maybe you should try a different description and use all lowercase for comments.
`if the first '--' isn't in PARSER or REMAINDER argument then take it out`

Reply:
This is a backport.